apps/dashboard/app/apps/dev_router.rb

```diff
@@ -25,26 +26,46 @@ def self.apps(owner: OodSupport::Process.user.name)
     end
   end
 
+  # Get the base path from with all development apps reside.
+  #
+  # @param [String] The name of the app owner (not used).
+  # @return [Pathname] The base path from with all development apps reside.
   def self.base_path(owner: OodSupport::Process.user.name)
     Configuration.dev_apps_root_path
   end
 
+  # Get the base path from with all development apps reside.
+  #
+  # @param [String] The name of the app owner (not used).
+  # @return [Pathname] The base path from with all development apps reside.
   def base_path
     self.class.base_path(owner: owner)
   end
 
+  # Get the type of router this is.
+  #
+  # @return [Symbol] The type of router this is.
   def type
     :dev
   end
 
+  # Get the relative URL for this router.
+  #
+  # @return [String] The relative URL for this router.
   def url
     "/pun/dev/#{name}"
   end
```

Comment:
I actually think that comments over one liners make it _harder_ to read, so I can't really tell if stuff like this is even useful.